phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexRebuildTaskIT.java

```diff
@@ -83,6 +82,7 @@ private String generateDDL(String format) {
     @Test
     public void testIndexRebuildTask() throws Throwable {
         String baseTable = generateUniqueName();
+        String viewName = generateUniqueName();
         Connection conn = null;
         Connection viewConn = null;
```

Comment:
nit: Could you please rename this to tenantConn, makes it more readable.

Reply:
done